### ISSUE TITLE
Fully adopt chef-dk (and drop Gemfile,...)

### ISSUE THREAD

**Issue by gildegoma** [gildegoma/chef-android-sdk#39]

**Comment by rjaros87** [comment]
Do you want install on Travis ChefDK instead of Ruby?

**Comment by gildegoma** [comment]
Yes, taking a direction like this example: https://github.com/chef-cookbooks/ark/blob/master/.travis.yml (using Kitchen against Docker containers, closing #28 and #5 by the way).
